apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -155,3 +155,25 @@ function wpcom_gutenberg_enable_custom_line_height() {
 	add_theme_support( 'custom-line-height' );
 }
 add_action( 'after_setup_theme', __NAMESPACE__ . '\wpcom_gutenberg_enable_custom_line_height' );
+
+/**
+ * Returns ISO 639 conforming locale string.
+ *
+ * @param string $language a language tag to be converted e.g. "en_US".
+ * @return string ISO 639 locale string e.g. "en"
+ */
+function get_iso_639_locale( $language ) {
+	$language = strtolower( $language );
+
+	if ( in_array( $language, array( 'pt_br', 'pt-br', 'zh_tw', 'zh-tw', 'zh_cn', 'zh-cn' ), true ) ) {
+		$language = str_replace( '_', '-', $language );
+	} else {
+		$language = preg_replace( '/([-_].*)$/i', '', $language );
```

Comment:
This might cause minor issues in Atomic for sites that set their language to, say, `zh_HK` for example.

https://wordpress.com/log-in/zh doesn't exist.

Though I tend to think it's a Calypso problem. We should be redirecting to English if we can't find a particular language.